src/config.rs

```diff
@@ -920,6 +923,34 @@ impl Config {
     }
 }
 
+fn parse_shell(raw: &String) -> Option<Vec<CString>> {
```

Comment:
Yeah the first argument is usually the executable. Anyway, my comment is invalid if we switch to std::command